src/containers/AdminContainer/AdminContainer.js

```diff
@@ -112,6 +113,38 @@ function AdminContainer(props) {
         postContext.Provider.updateSelectedPost(post)
     }
 
+    const validatePostDetails = () => {
```

Comment:
Can we define constants instead of using magic numbers? assuming we're currently checking for valid lat long in the san jose region?

Reply:
Ah good point yes will likely define the magic numbers as boundries tonight. good catch